opendevin/server/session.py

```diff
@@ -117,7 +117,10 @@ async def start_task(self, start_event):
         if self.controller is None:
             await self.send_error("No agent started. Please wait a second...")
             return
-        self.agent_task = asyncio.create_task(self.controller.start_loop(task), name="agent loop")
+        try:
+            self.agent_task = await asyncio.create_task(self.controller.start_loop(task), name="agent loop")
+        except Exception:
```

Comment:
I don't _think_ this except will ever fire, because it's running async

Reply:
Yeah I had to look that one up.  Apparently it won't fire UNLESS you `await` the coroutine because exceptions are only returned to the calling function if it's `await`ed.